### ISSUE TITLE
smudgeplot.py hetkmers -k option missing

### ISSUE THREAD

**Issue by pwkooij** [KamilSJaron/smudgeplot#48]
I'm trying to run smudgeplot on my data but get stuck with the following line of code I got from the manual [https://github.com/KamilSJaron/smudgeplot/wiki/manual-of-smudgeplot-with-jellyfish](url)

`jellyfish dump -c -L $L -U $U kmer_counts.jf | smudgeplot.py hetkmers -k 21 -o kmer_pairs`

It seems there is no option -k. I receive the following output:
``usage: smudgeplot hetkmers [-h] [-o O] [--middle] [infile]
smudgeplot hetkmers: error: argument infile: can't open '21': [Errno 2] No such file or directory: '21'``

When I go into the help of smudgeplot I see this:

```
smudgeplot.py hetkmers -h
usage: smudgeplot hetkmers [-h] [-o O] [--middle] [infile]

Calculate unique kmer pairs from a Jellyfish or KMC dump file.

positional arguments:
  infile      Alphabetically sorted Jellyfish or KMC dump file (stdin).

optional arguments:
  -h, --help  show this help message and exit
  -o O        The pattern used to name the output (kmerpairs).
  --middle    Get all kmer pairs that are exactly the same but in the middle
              nt. When this flag is used, the input dump must be
              alphabetically sorted/ (default: different by a SNP at any
              position).
``` 

I'm running smudgeplot v0.2.1

**Comment by KamilSJaron** [comment]
Hi there, the parameter `k` was removed as useless, now we just count number of letters in the first processed kmer (we figured we don't have to ask user to mention it explicitly). Hence something like

```
jellyfish dump -c -L $L -U $U kmer_counts.jf | smudgeplot.py hetkmers -o kmer_pairs
```

should work.

I wonder if we have somewhere a forgotten piece of documentation that still uses `k`. Where did you find the information about `k`?
